tasks/main.yml

```diff
@@ -7,8 +7,9 @@
 - name: Ensure that mutually exclusive options are not set
   assert:
     that:
-      - not (galaxy_manage_clone and galaxy_manage_download)
-    fail_msg: "Variables galaxy_manage_clone and galaxy_manage_download cannot both be true."
+      - [galaxy_manage_clone | bool, galaxy_manage_download | bool, galaxy_manage_existing | bool].count(True) == 1
```

Comment:
@natefoo This syntax works ok in newer ansible versions, but it looks like it doesn't on ansible<2.8 on travis. Any thoughts on an alternative?

Reply:
Not the most elegant, and I might have missed something cause my logic is rusty, but how about:
```
(not (galaxy_manage_clone and (galaxy_manage_download or galaxy_manage_existing))) and (not (galaxy_manage_download and (galaxy_manage_clone or galaxy_manage_existing))) 
```
I think that should cover all possible pairs?

Reply:
I'm fine with excluding <2.8, we keep these pretty updated, people should be updating their ansible, and this feels cleaner and easier to keep updated.